aiven/client/cli.py

```diff
@@ -1187,11 +1187,35 @@ def service__privatelink__availability(self):
 
     @arg.project
     @arg("--format", help="Format string for output")
+    @arg("--limit", help="Limit number of results showing in each page.", type=int)
+    @arg("--page", help="Page number.", type=int, default=0)
```

Comment:
Hmm, I'd say that we shouldn't implement our own paging support here but just output everything. If somebody need to see the result page by page they can pipe it to an external pager like `more` or `less`.

Moreover those `PAGE n/m` headers currently break automatic parsing of the result.